tests/formats/test_mrea.py

```diff
@@ -45,6 +47,10 @@ def compare_all_instances(
     decoded2 = check_all_instances(encoded, asset_manager.target_game, all_instances)
 
     assert test_lib.purge_hidden(decoded2.raw) == test_lib.purge_hidden(decoded.raw)
+    if simple_construct is not None:
```

Comment:
a comment/docstring indicating that this param is used for bytematching would be helpful, cuz it isn't super obvious from the name imo